test/forms/declaration/DeclarantDetailsSpec.scala

```diff
@@ -16,38 +16,54 @@
 
 package forms.declaration
 
+import models.DeclarationType._
 import org.scalatest.{MustMatchers, WordSpec}
 import play.api.libs.json.{JsObject, JsValue}
 
 class DeclarantDetailsSpec extends WordSpec with MustMatchers {
 
   import DeclarantDetailsSpec._
 
-  "DeclarantDetails mapping used for binding data" should {
+  Seq(SUPPLEMENTARY, STANDARD, OCCASIONAL, SIMPLIFIED).map { decType =>
```

Comment:
New test for CLEARANCE, empty form, no errors?

Reply:
Good point! Thanks. I'll add this

Reply:
Added